2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+Konkrete Quellen von Ressourcen:
+
+- Arbeit an Freier Software als Hobby (Sinnstiftung durch Problemlösen, Zusammenarbeit mit anderen, Anerkennung durch positives Feedback, Lerneffekt, Demonstration von Fähigkeiten (z.B. für Bewerbungsverfahren))
+- Durch Unternehmen oder Behörden bezahlte Entwicklung (motiviert durch Eigeninteresse (Einsicht in die Vorteile))
+- Spenden (Altruismus oder aufgeklärtes Eigeninteresse)
+- Tragfähige [Geschäftsmodelle](https://en.wikipedia.org/wiki/Business_models_for_open-source_software)
+
+Fakt ist aber: in den allermeisten Projekten fehlt es an Ressourcen für eine wünschenswerte Weiterentwicklung.
+Mit anderen Worten: obige Quellen von Ressourcen sind nicht ausreichend.
+
+
+## Analyse existierender Geschäftsmodelle
+
+Wikipedia listet folgende [Geschäftsmodelle](https://en.wikipedia.org/wiki/Business_models_for_open-source_software)
+für "Open-Source-Software" auf:
+
+    3.1 Dual-licensing
+    3.2 Selling professional services
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.5 Selling software as a service
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+    3.12 Selling of optional proprietary extensions
+    3.13 Selling of required proprietary parts of a software product
+    3.14 Selling of proprietary update systems
+    3.15 Re-licensing under a proprietary license
+    3.16 Obfuscation of source code
+    3.17 Delayed open-sourcing
+    3.18 Open sourcing on end-of-life
+
+
+ Für die Projekte mit der Zielgruppe der Endbenutzer\*innen kommen bestimmte Modelle
+ nicht in Frage bzw. sind nicht wünschenswert (Unterschied zwischen Open-Source und freier Software).
+ Im Folgenden sind Ausschlussgründe und die davon betroffenen Geschäftsmodelle aufgelistet.
+
+
+- Nicht sinnvoll, Weil kein Weiterverkauf angestrebt wird
+    - Dual-licensing
+    - Re-licensing under a proprietary license
+- Zielkonflikt: Perfekte Software braucht keinen Support/Schulung
+    - Selling professional services
+- Problem der Datensouveränität.
+    - Selling software as a service
+
+
+- Widerspricht dem Ziel maximalen FLOSS-Anteils
+    - Selling of optional proprietary extensions
+    - Selling of required proprietary parts of a software product
+    - Selling of proprietary update systems
+    - Obfuscation of source code
+    - (Delayed open-sourcing)
+
+- Nicht nachhaltig
+    - Open sourcing on end-of-life
+
+Es bleiben also folgende Modelle übrig:
+
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
```

Comment:
Kommerzielle Entwicklung die zur Erweiterung der Community-Edition führt (IMHO Hauptgeschäftsmodell bei Conversations)

auch: Sponsoring – Zahlung für Werbung oder andere Gegenleistung

Reply:
Proprietäre Software (auch in Form von Erweiterungen) sehe ich nicht als speziell Förderungswürdig an. Dafür gibt es den etablierten Softwaremarkt.


Sponsoring ist halt ein ganz heißes Eisen. Ich rechne damit, dass es viele nicht wollen. Außerdem kommen Fragen auf, welche Werbung man zulässt etc. Was ich mir vorstellen könnte, ist "Werbung in eigener Sache" (Projekt- oder FFS-bezogen) oder kleine Nerv-Elemente, die einen latenten Anreiz geben, ein Support-Zertifikat zu kaufen. So ähnlich wie bei Shareware früher.